python_scripts/one_time_scripts/update_agentids.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/python3
+# This script takes an Excel file, "combined-aspace-agents-edited.xlsx", and uses the "combined and cleaned" sheet,
+# takes the agent link from the "Aspace_link" column, grabs the agent JSON from ArchivesSpace using the API, then adds
+# a new Record ID to the agent, if an existing record ID does not already exist. Non-matching record IDs are logged and
+# the one existing in ArchivesSpace remains. Record IDs are located as columns in the sheet, named "Wikidata_id",
+# "SNAC_id", "LCNAF_id", "ULAN_id", "VIAF_id", and "local". After adding the Record ID to the JSON locally, the script
+# sorts the order of the IDs according to the above and posts the updated agent record via the ArchivesSpace API.
+import argparse
+import os
+import pandas  # TODO: update the Wiki for this script - secrets.py file will not work as will cause ImportError: cannot import name randbits when loading pandas. See https://stackoverflow.com/questions/73055157/what-does-importerror-cannot-import-name-randbits-mean
```

Comment:
Has this todo been done?  Do we need to file a follow on task for it?

Reply:
I just wrote a note about that particular obstacle here: [Script Descriptions - update_agentids.py](https://github.com/Smithsonian/caas-aspace-scripts/wiki/Script-Descriptions#update_agentidspy). I think we'll need to have a conversation about how to handle secrets.py moving forward if we plan on using pandas in other scripts. I can make a card for that on our board.